discovery-provider/src/tasks/index_rewards_manager.py

```diff
@@ -113,25 +113,29 @@ def cache_latest_sol_rewards_manager_db_tx(redis: Redis, latest_tx):
     cache_latest_sol_db_tx(redis, latest_sol_rewards_manager_db_tx_key, latest_tx)
 
 
+CHALLENGE_TYPE_MAP: Dict[str, TransactionType] = {}
```

Comment:
very nitty but typically we usually all caps for constant values, and technically this one changes bc it's assigned after get_challenge_type_map

Reply:
agree, was getting lint error but will disable.